### ISSUE TITLE
Geomap: Table Filter Focus Issue

### ISSUE THREAD

**Issue by upsonp** [wet-boew/wet-boew#7996]
Hi,

I'm struggling to figure out how I'm going to describe this problem. Underneath the map on a page using the GeoMap plugin there's normally a table. At the top of the table there's a filter box. The issue I'm having is when you start typing in the filter box it seems to lose focus, so you can only enter one character at a time. I thought it was just an issue with my implementation, but after trying it out on the GeoMap example page I was able to reproduce it. I tried to look up the issue, but I didn't find anyone who's already reported it.

Steps to reproduce:
1) Go to the demo page. http://wet-boew.github.io/wet-boew/demos/geomap/geomap-en.html
2) Turn on the "KML Demo" layer
3) Below the map select the "KML Demo" tab
4) Start typing "Metal" in the filter box.

Notice after hitting the 'M' the box loses focus. Then you click back in the box and push 'e' and again the box loses focus.

Is this just me?

**Comment by crochefort** [comment]
Nope just try with Chrome 58.0.3029.96 (64-bit) and got the same result... 
Same thing with Firefox 53.0 (64-bit)

**Comment by jvanulde** [comment]
This has been resolved with PR: https://github.com/wet-boew/wet-boew/pull/7889

**Comment by lucas-hay** [comment]
@jvanulde , Have you looked at #7889 comments lately?
We have requested a couple changes in that PR.

**Comment by jvanulde** [comment]
@lucas-hay yes I have. I will need to find time to address those at some point.

**Comment by kodecount** [comment]
I don't think the PR well truly fix the focus issues with tables. This is a much large issue. See my comment in https://github.com/wet-boew/wet-boew/issues/8019.
